### DISCUSSION TITLE
Load image stored in device using uri in Compose Multiplatform?

### DISCUSSION THREAD

**Discussion by BitByAVampire** [coil-kt/coil#2582] [Q&A] [answered]
I tried

```
AsyncImage(
                model = ${URI to Image here},
                contentDescription = null,
                contentScale = ContentScale.Fit)
```
but i did not see the image loading.  
I also tried to use the ImageBuilder with an ImageRequest, still did not work. 
I am asking how (if its even possible) to load an image given an uri for an image stored in device. (either android or ios)

**Comment by colinrtwhite** [comment, marked as answer]
I'd take a look at the `onError` param in `AsyncImage` to debug your exception. Local image URIs should be supported.
